pyproject.toml

```diff
@@ -41,6 +41,7 @@ markdownify = "^0.13.1"
 click = "^8.1.7"
 google-generativeai = "^0.8.3"
 markdown2 = "^2.5.2"
+python-magic = "^0.4.27"
```

Comment:
is it possible to make this an optional dependency? Companies that may want surya may not want a large list of dependencies for features they don't use, as that makes maintenance harder.